spyglass.py

```diff
@@ -438,8 +438,12 @@ def sanitize(string: str) -> Optional[str]:
 logger.info("Spreadsheet populated!")
 print("Saving spreadsheet...")
 
-outfile_path = Path(args.outfile)
-outfile_path.parent.mkdir(parents=True, exist_ok=True)
+try:
+    outfile_path = Path(args.outfile)
+    outfile_path.parent.mkdir(parents=True, exist_ok=True)
+except FileExistsError: # if outfile_path.parent currently exists as a file
+    logger.error(f"{outfile_path.parent} already exists as a file.")
+    print(f"Could not save spreadsheet: {outfile_path.parent} already exists as a file.")
```

Comment:
Oh, right, my bad. Now that you bring it up, it's probably also better to fail upfront, rather than waiting until after the dump is downloaded and parsed and a .xlsx representation is built in memory.

Reply:
That's probably best, yes.